src/services/music-providers/yandex-music.service.ts

```diff
@@ -54,4 +65,16 @@ export class YandexMusicService extends BaseMusicService {
     ): Promise<void> {
         throw new Error('Method not implemented.');
     }
+
+    private getAllArtistsNameFromArtistObject(artist: Artist) {
+        if (!artist?.decomposed) {
+            return [artist.name];
+        }
+
+        const artistsInDecomposed = artist.decomposed.filter(
```

Comment:
Please consider renaming `artistsInDecomposed` to a more indicative name like `filteredArtists`. It enhances code readability, making it easier to understand the purpose of the variable.